pkg/healthcheck/microshift_core_workloads.go

```diff
@@ -30,15 +32,76 @@ func getCoreMicroShiftWorkloads() (map[string]NamespaceWorkloads, error) {
 			},
 		},
 	}
-	fillOptionalWorkloadsIfApplicable(cfg, workloads)
+	if cfg.Network.IsEnabled() {
+		workloads["openshift-ovn-kubernetes"] = NamespaceWorkloads{
+			DaemonSets: []string{"ovnkube-master", "ovnkube-node"},
+		}
+	}
+	if err := fillOptionalWorkloadsIfApplicable(cfg, workloads); err != nil {
+		return nil, err
+	}
 
 	return workloads, nil
 }
 
-func fillOptionalWorkloadsIfApplicable(cfg *config.Config, workloads map[string]NamespaceWorkloads) {
-	klog.V(2).Infof("Configured storage driver value: %q", string(cfg.Storage.Driver))
-	if cfg.Storage.IsEnabled() {
-		klog.Infof("LVMS is enabled")
+func lvmsIsExpected(cfg *config.Config) (bool, error) {
+	cfgFile := "/etc/microshift/lvmd.yaml"
+	if exists, err := util.PathExists(cfgFile); err != nil {
+		return false, err
+	} else if exists {
+		klog.Infof("%s exists - expecting LVMS to be deployed", cfgFile)
+		return true, nil
+	}
+
+	if !cfg.Storage.IsEnabled() {
+		klog.Infof("LVMS is disabled via config. Configured value: %q", string(cfg.Storage.Driver))
+		return false, nil
+	}
+
+	cmd := exec.Command("vgs", "--readonly", "--options=name", "--reportformat=json")
```

Comment:
Could we simplify the code by running the following command? 
`vgs --readonly --noheadings --options=name`

It's just a list of names then, so no need for json parsing.

Reply:
But the output of vgs is lines with extra whitespace. So we'd have to split on new lines then remove the whitespace. So one way or another we need to do some processing

Reply:
Is there a way to do this natively in go? Does lvms do something similar to check it out?

Reply:
Patryk provided the [source](https://github.com/openshift/lvm-operator/blob/89234f2d32e1df7bf7dc6326f09fa68407d235e1/internal/controllers/vgmanager/lvm/lvm.go#L348-L395) from lvms but its in the internal libraries.
This is lgtm to me.